echoshader/dev/echo_gram.py

```diff
@@ -1,57 +1,179 @@
-from echo_lib import *
+"""echo_gram.py module
 
+use panel to get a echogram with control widgets
+"""
+import panel
 
-def simpleEchogram(
-    ds_Sv, clim=(-80, -30), cmap="jet", rasterize=True, width=600, height=300
-):
-    """
-    input echopype MVBS DataSet / Sv DataArray to get a simple echogram
-    """
+panel.extension("vtk")
+
+import warnings
 
-    plot = ds_Sv.hvplot(
-        kind="image",
-        x="ping_time",
-        y="echo_range",
-        c="Sv",
-        title="Echogram",
-        cmap=cmap,
-        clim=clim,
-        rasterize=rasterize,
-        width=width,
-        height=height,
-        flip_yaxis=True,
-    )
+import echo_map
+import echo_stats
+import hvplot.xarray
+import pandas
+import param
 
-    return plot
+warnings.simplefilter("ignore")
 
 
 def echogram(MVBS):
+    """Get an echogram with control widgets
+
+    Parameters
+    ----------
+    MVBS : xr.Dataset
+        MVBS_ds data set or ds_Sv data array
+
+    Returns
+    -------
+    type:
+        param class
+    describe:
+        return values contains echogram and control widgets
+        use 'panel.Row(echogram(MVBS).param,echogram(MVBS).view)'
+        to get a complete panel
+        echogram(MVBS).param shows param(control widgets)
+        echogram(MVBS).view shows echogram image
```

Comment:
```suggestion
   param.Parameterized
        return values contains echogram and control widgets
        use 'panel.Row(echogram(MVBS).param,echogram(MVBS).view)'
        to get a complete panel
        echogram(MVBS).param shows param(control widgets)
        echogram(MVBS).view shows echogram image
```

Same comment as above here regarding following the numpy returns docstring. Please see numpy documentation for proper convention for this.